ci/common.sh

```diff
@@ -0,0 +1,12 @@
+# Common material sourced by CI scripts in this directory
```

Comment:
Should this one have some kind of #! at the beginning - to make sure it uses the correct interpreter?

Reply:
I was thinking that by leaving it out we indicate that it's supposed to be sourced in, not run stand-alone.

Reply:
well, that's fair.